pkgs/development/compilers/swift/default.nix

```diff
@@ -24,12 +27,26 @@ let
     # used in Swift, and applies the same libc overrides as `apple_sdk.stdenv`.
     clang = if pkgs.stdenv.isDarwin
       then
-        llvmPackages_15.clang.override rec {
+        let
+        swiftLlvmPackages.clang.override rec {
           libc = apple_sdk.Libsystem;
           bintools = pkgs.bintools.override { inherit libc; };
+          # Ensure that Swift’s internal clang uses the same libc++ and libc++abi as the
+          # default Darwin stdenv. Using the default libc++ avoids issues (such as crashes)
+          # that can happen when a Swift application dynamically links different versions
+          # of libc++ and libc++abi than libraries it links are using.
+          inherit (pkgs.llvmPackages) libcxx;
+          extraPackages = [
+            pkgs.llvmPackages.libcxxabi
+            # Use the compiler-rt associated with clang, but use the libc++abi from the stdenv
+            # to avoid linking against two different versions (for the same reasons as above).
+            (pkgs.swiftLlvmPackages.compiler-rt.override {
```

Comment:
pkgs.swiftLlvmPackages => swiftLlvmPackages

Reply:
I dropped `pkgs` from all the llvmPackages stuff since it wasn’t necessary for that.